pollination/ladybug_comfort/map.py

```diff
@@ -291,14 +291,24 @@ class UtciMap(Function):
         'will be for the entire result_sql run period.', default=''
     )
 
+    plain_text = Inputs.str(
```

Comment:
Maybe we should call this `output_format` since that's a better description of what this pollination function is.

Reply:
I changed the name of the function input.